packages/bbui/package.json

```diff
@@ -14,17 +14,12 @@
     "./spectrum-icons-vite.js": "./src/spectrum-icons-vite.js"
   },
   "scripts": {
-    "build": "rollup -c"
+    "build": "vite build"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^16.0.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^11.2.1",
+    "@sveltejs/vite-plugin-svelte": "1.4.0",
     "postcss": "^8.2.9",
```

Comment:
We can remove the postcss plugin if we don't need it in the vite config as mentioned above.